src/TimeGrid.jsx

```diff
@@ -44,7 +44,9 @@ let TimeGrid = React.createClass({
     return {
       step: 30,
       min: dates.startOf(new Date(), 'day'),
-      max: dates.endOf(new Date(), 'day')
+      max: dates.endOf(new Date(), 'day'),
+      type: 'gutter',
+      now: new Date()
```

Comment:
this is a special case: I was using ...DaySlot.defaultProps and there was some weird stuff going down.  Occasionally it just crashed, so I removed the spread and pulled in the defaults we really needed.  Probably an obscure React bug